packages/javascript-sdk/src/oauth2-client/interfaces.ts

```diff
@@ -40,6 +40,7 @@ interface GetAuthorizationUrlOptions extends ConfigOptions {
   state?: string;
   verifier?: string;
   query?: StringDict<string>;
+  prompt?: 'none' | 'login' | 'consent';
```

Comment:
My understanding is that the requirement in the ticket was for `none` to mean that `?prompt=none` is passed in the `authorize` call, `login` and would be used for centralized login. Maybe a silly question, but what does `consent` mean here - what is the consequence of specifying this option?

Reply:
This is a good point. These three values for the `/authorize` endpoint are valid, but we are likely unable to support the `'consent'` option. Should remove this option as to not confuse the dev?

Reply:
I think this is misleading if it doesn't actually do anything or isn't compatible in the context of the SDK and should be removed. I'd worry that it might inadvertently trick developers into thinking some action should occur if they specify consent, for example showing a consent collection page.